### DISCUSSION TITLE
Skin priority, ability to use 2 and more skins at once

### DISCUSSION THREAD

**Discussion by Sarisan** [ppy/osu#23737] [Ideas]
You can choose two skins at once, skin on the top of chosen list has the higher priority. If it misses some assets like for mania, it looks for assets in lower priority skin. If no game's default skin found in chosen list it gets the lowest priority.

**Comment by peppy** [comment]
Originally we were looking at allowing a skin choice per ruleset, would that also satisfy your issue?

**Reply by Sarisan** [reply]
> Originally we were looking at allowing a skin choice per ruleset, would that also satisfy your issue?

Well, probably.

**Reply by Clozent** [reply]
I think skin priority should be a thing that comes alongside a setting for every mode. This way, for example, one could use a skin that contains only hitcircles, and put another skin in a lower priority, so they would essentially be using the lower priority skin, just with some elements replaced.

**Reply by nagi-desuuu** [reply]
Like what Danser is doing with their Primary and Fallback skins?
